app/services/pii/signing_key.rb

```diff
@@ -0,0 +1,32 @@
+module Pii
+  class SigningKey
+    include Encodable
+
+    attr_accessor :rsa_key
+
+    DIGEST = 'SHA256'.freeze
+    @rsa_key = nil
```

Comment:
I don't think that the class-level `@rsa_key` is being used since it's only available in class/static methods, of which there are none? The `attr_accesor` is instance methods. (you can do `attr_accessor` inside of a `class << self` block to define accessors on the class....but I don't think we need those)

I would do this:

```ruby
def initialize
  # set the instance variable from the class method
  self.rsa_key = self.class.pii_signing_key
end

def self.pii_signing_key
  # static method, so this is the class variable not the instance variable
  @rsa_key ||= OpenSSL::PKey::RSA.new(pii_signing_key_pem, Figaro.env.pii_signing_passphrase)
end

def self.pii_signing_key_pem
  File.read(Rails.root + 'keys/pii_signing.key.enc')
end
```